src/sentry/eventstream/base.py

```diff
@@ -43,13 +46,28 @@ def _dispatch_post_process_group_task(
         if skip_consume:
             logger.info("post_process.skip.raw_event", extra={"event_id": event.event_id})
         else:
-            post_process_group.delay(
-                event=event,
-                is_new=is_new,
-                is_regression=is_regression,
-                is_new_group_environment=is_new_group_environment,
-                primary_hash=primary_hash,
-            )
+            random_val = random.random()
```

Comment:
Out of curiousity, what's this random for?

Reply:
I wanted to have the new code path be used incrementally as the option value is increased from 0.0 to 1.0 and thought that this would be a low overhead way to do that.